### ISSUE TITLE
Alt attributes need to be included for all graphical elements (visualization and images)

### ISSUE THREAD

**Issue by philipashlock** [GSA/datagov-wptheme#84]
Alt attributes need to be included for all graphical elements (visualization and images)

**Comment by philipashlock** [comment]
The only images should be in the content on pages and posts which can be updated by the editor. If any of these posts are missing an alt tag we can ticket it specifically.
